### ISSUE TITLE
Could canvasXpress respect gridlines in ggplot2 better?

### ISSUE THREAD

**Issue by davidhodge931** [neuhausi/canvasXpress#148]
Particularly: 
* know which major gridlines are blank
* know the linewidth of gridlines present

```
library(tidyverse)
library(palmerpenguins)
library(canvasXpress)

p <- penguins |> 
  ggplot() +
  geom_point(aes(x = flipper_length_mm, 
                 y = body_mass_g, 
                 colour = species,)) +
  theme(panel.grid = ggplot2::element_line(linewidth = 5)) +
  theme(panel.grid.major.x = element_blank()) +
  theme(panel.grid.minor = element_blank()) 

p

canvasXpress(p)
```
![image](https://github.com/neuhausi/canvasXpress/assets/55065152/729ad7f0-10d7-4cbd-95da-0533dcd9032a)

**Comment by neuhausi** [comment]
Fixed in version 48.6 too. The units are not honored as they will scale according to the size of the plot.
![cX-htmlwidget-2a56c0198d2ebf914823-cx](https://github.com/neuhausi/canvasXpress/assets/254923/64d10729-6cb5-4fae-a000-43f74caf133f)

**Comment by davidhodge931** [comment]
Works great :)
